### PR TITLE
2741 redirect c course page to new find

### PR DESCRIPTION
**Pull request by defong**
### Context
Flanging rails redirect

### Changes proposed in this pull request
Added the flag to redirect it

### Guidance to review
Like @dankmitchell but not railsy...

### CODE REVIEW

**Comment by dankmitchell** [comment]
@defong @timabell  set the environment variable on the Heroku review app - https://dashboard.heroku.com/apps/search-and-compare-ui-pr-486/settings

But it looks like we have a re-direct loop?

**Comment by defong** [comment]
@rizzkhan1 

new env variable
FEATURE_REDIRECT_TO_RAILS
NEW_APP_URL

**Comment by rizzkhan1** [comment]
> @rizzkhan1
> 
> new env variable
> FEATURE_REDIRECT_TO_RAILS
> NEW_APP_URL
NEW_APP_URL is already there, just  add FEATURE_REDIRECT_TO_RAILS to azure/template.json parameters and app settings.

File: azure/template.json [hunk 1]
```diff
@@ -93,6 +93,13 @@
             "metadata": {
                 "description": "Base URL of new FIND"
             }
+        },
+        "featureRedirectToRails": {
+            "type": "bool",
+            "defaultValue": {},
```
Comment (rizzkhan1) [inline azure/template.json]: default value should be either true or false (without quotes or brackets) for boolean type variables

Comment (dankmitchell) [reply azure/template.json]: Default would be `false`

**Pull request merged**